### ISSUE TITLE
[Suggestion] add open computers support to mechanical multiblock

### ISSUE THREAD

**Issue by hollowhalls** [malte0811/IndustrialWires#78]
would you consider adding open computers methods for multiblock speed, energy stored, and possibly commutator efficiency for the mechanical mutiblock.

**Comment by anar4732** [comment]
did you tried to connect adaptor to redstone block?

**Comment by hollowhalls** [comment]
yes, opencomputers console did not recognize anything connected to the adapter

**Comment by CassandraRC7** [comment]
If you set some constants, like the moment of inertia and the max speed, you can get something close to good OpenComputers control, just by using the redstone signal.

**Comment by Meserion** [comment]
To prevent the mechanical multiblock from blowing up, I use a comparator set to subtract mode and subtract a signal of "14" from it's speedometer. Make sure to set the Speedometer to near its max speed that your flywheel can handle; when it gets to the max it will send a 15 strength redstone signal. Use that redstone signal combined with redstone wires to turn off or else disengage any power production systems to prevent Flywheel explosions. Very simple to setup, no computers needed. The Logarithmic side I could see having a computer hookup to measure how full the multiblock is, but industrial wires sorta of has something in the form of its control panel with a meter connection.
